styles.css

```diff
@@ -699,22 +702,29 @@
     border-color: #36b37e;
   }
 
-  .desktop-nav {
+  .desktop-nav,
+  .in-touch,
+  .illustration-desktop,
+  hr {
     display: none;
   }
 }
 
 @media screen and (min-width: 1000px) {
   .top-bar,
   .space,
-  .indicator {
+  .indicator,
+  .illustration-contact,
+  .touch-box,
+  .footer {
```

Comment:
[Optional] Kindly adjust your media query to include screen sizes below 900px.

Here's how the website looks below 1000px:

![image](https://user-images.githubusercontent.com/20554583/193247518-20bdb765-dbb5-4b9e-9519-bc10686a0174.png)